README.md

```diff
@@ -25,6 +25,11 @@ Add users and roles:
 sh addUsersAndRoles.sh
 ```
 
+Add the spreadsheet:
+```
+./bin/ccd-import-definition.sh ~/fpl-ccd-configuration/FPL_CCD_DEFINITION_TEMPLATEx.xlsx
```

Comment:
A `./bin/ccd-import-definition.sh` does not exist in project directory. Please correct path. Also `~/fpl-ccd-configuration/` part won't work for everyone.

Reply:
./ccd-docker/bin ...